buildSrc/src/main/java/AndroidComponents.kt

```diff
@@ -3,5 +3,5 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 object AndroidComponents {
-    const val VERSION = "54.0.20200818130156"
+    const val VERSION = "56.0.20200819190136"
```

Comment:
How did we skip 55? 🤔

Reply:
Is there a bug in the script? The PR merged yesterday bumped to a 54 version. https://github.com/mozilla-mobile/fenix/commit/5f5caa4720c1f18637ba12946204e8f162da7b83

Or maybe whoever did the previous release forgot to bump the version number to 55 after they did the 54 cut?

Reply:
Ah, it was an error in the release. We'll have to document that 55 was skipped in error and maybe write a release script check to ensure the increments are exactly i++.